go.mod

```diff
@@ -7,6 +7,8 @@ replace (
 	github.com/blizzy78/varnamelen => github.com/blizzy78/varnamelen v0.8.0
 	github.com/coreos/etcd => github.com/coreos/etcd v3.3.24+incompatible
 	github.com/dgrijalva/jwt-go => github.com/dgrijalva/jwt-go v3.2.1-0.20210802184156-9742bd7fca1c+incompatible
+	// To fix: [CVE-2024-37298] CWE-770: Allocation of Resources Without Limits or Throttling
+	github.com/gorilla/schema => github.com/gorilla/schema v1.4.1
```

Comment:
We should try to avoid the replace statements now, this app should now be upgraded to go version 1.23 and this replace section removed.